public/js/collections/trips.js

```diff
@@ -1,9 +1,22 @@
 'use strict';
 
 var _ = require( 'lodash' );
-var Backbone = require( 'backbone' );
+var Collection = require( 'ampersand-rest-collection' );
+var LineModel = require( '../models/line' );
+var TripModel = require( '../models/trip' );
 
-var TripsCollection = Backbone.Collection.extend({
+var TripsCollection = Collection.extend({
+
+  props: {
+    lastRefreshed: 'date',
+    loaded: 'boolean'
+  },
+
+  children: {
+    line: LineModel
+  },
```

Comment:
May be best to have "line" be a prop, so that it doesn't get re-instantiated. I'm really not sure I like that part of the `children` behavior.